schema.sql

```diff
@@ -1,5 +1,10 @@
 /* Database schema to keep the structure of entire database. */
 
-CREATE TABLE animals (
-    name varchar(100)
-);
+create table animals ( 
+    id int, 
```

Comment:
- [OPTIONAL] It will be more professional if you make the Id column not null and autoincremented 👌🏽. E.g
```
    id int NOT NULL IDENTITY(1,1)
```
- [OPTIONAL] you can also mark the id column as primary key by adding 
```
    PRIMARY KEY (id)
```

Reply:
HI @rotimiazeez 👋
Thanks for reviewing and suggesting changes in my PR.
Can you please guide me on how to update the Id column?